### ISSUE TITLE
Tag spacing messed up when used with Twitter Bootstrap

### ISSUE THREAD

**Issue by rsimon** [lucaong/jQCloud#24]
Hi! Thanks for jQCloud - works like a charm!

However, I used it with Twitter Bootstrap (http://twitter.github.com/bootstrap/), which messed up the tag layout. Tags were placed very closely, so that they overlapped heavily. The cause seemed to be that the Bootstrap CSS sets a global line-height property in the body.

I added a "line-height:normal" to the div.jqcloud CSS class, and this fixed it.

In other words: my problem is solved, but I just wanted to let you know in case you want to add that as well!

Cheers,
Rainer
